languages/common-lisp/reference/README.md

```diff
@@ -0,0 +1,84 @@
+# Common Lisp Reference
+
+Please note that this is a *very* WIP document. I'm currently using [this
+document][csharp-example] as an example to base my work off of.
+
+## Concepts
+### General
+  - Constants
+  - Variables
+  - Strings
+  - Characters
+  - Functions
+  - Sameness
+  - Comments
+  - Nested Functions
+  - Conditionals
+  - Expressions
+  - Map
+  - Generic Setters
+  - Enumeration
+  - Recursion
+  - Higher Order Functions
+  - Anonymous Functions
+  - String Formatting
```

Comment:
A sub-cateogry of `Strings` perhaps?

Reply:
I've realized that I already have a section for Format, so I've removed this and just left Strings under types.